sda/export_test.go

```diff
@@ -48,25 +42,35 @@ func (o *Overlay) TokenToNode(tok *Token) (*TreeNodeInstance, bool) {
 	return tni, ok
 }
 
-func (h *Host) AbortConnections() error {
-	h.closeConnections()
-	close(h.ProcessMessagesQuit)
-	return h.host.Close()
+// AddTree registers the given Tree struct in the underlying overlay.
+// Useful for unit-testing only.
+func (h *Host) AddTree(t *Tree) {
+	h.overlay.RegisterTree(t)
 }
 
-func (h *Host) CloseConnections() error {
-	return h.closeConnections()
+// AddRoster registers the given Roster in the underlying overlay.
+// Useful for unit-testing only.
+func (h *Host) AddRoster(el *Roster) {
+	h.overlay.RegisterRoster(el)
 }
 
-func (h *Host) RegisterConnection(e *network.ServerIdentity, c network.SecureConn) {
-	h.networkLock.Lock()
-	defer h.networkLock.Unlock()
-	h.connections[e.ID] = c
+func (t *TCPRouter) AbortConnections() error {
```

Comment:
No `Close` is already defined by `Router` , this was to simulate a hard network failure of a node but in fact it does not do what we want. I kept this a compatibility in `export_test.go` since I'm sure @ineiti would want to use it, but I don't see how. For more info see `TestTcpRouterReconnection`.